### PR TITLE
Atmnlev fix

### LINKED ISSUE #786: Bug when atmnlev NOT NONE
When running test ERP_D_Ln9.T42z30_T42.FDABIP04.hobart_nag.cam-outfrq9s, getting the following error message.

  File "/home/fischer/code/cesm2_0_alpha03c/cime/scripts/Tools/../../utils/python/CIME/XML/grids.py", line 150, in _get_gridmaps
    grids[0] += "z"+atmnlev
TypeError: can only concatenate tuple (not "str") to tuple

It looks like the only problem is a missing comma at the end of the line since it's appending to a tuple.  Also line 152 has the same problem.

           grids[1] += "z"+lndnlev

I'm not sure if this is the fix, since I'm not familiar with this section of code.

This is a bug that @cacraigucar passed back to me.

### PR DESCRIPTION
**Pull request by fischer-ncar**
Fix bug where T42z30_T42 grid was causing an error when concatenating tuples.

Test suite:   scripts_regression_tests.py
Test baseline: 
Test namelist changes: 
Test status: 

Fixes #786 

User interface changes?: 

Code review:

### CODE REVIEW

**Comment by jgfouca** [comment]
We probably don't want to see upstream merges in your PR. Consider rebasing instead of merging when updating your branch.

File: utils/python/CIME/XML/grids.py [hunk 1]
```diff
@@ -147,9 +147,9 @@ def _get_gridmaps(self, component_grids, atmnlev=None, lndnlev=None):
                             logger.debug(" %s: %s" %gridmap)
 
         if atmnlev is not None:
-            grids[0] += "z"+atmnlev
+            grids[0] += "z"+atmnlev,
```
Comment (jgfouca) [inline utils/python/CIME/XML/grids.py]: I think this would be more clear:

    grids[0][0] += "z"+atmnlev

Comment (jgfouca) [inline utils/python/CIME/XML/grids.py]: So if these tuples need to be mutable, then they should be lists, not tuples.

**Comment by fischer-ncar** [comment]
I'm closing this pull request since this fix no longer works with new the config_grids.  I'll need to revisit this with an updated fix.

**Pull request closed without merging**